File: cs/proxy/dice/GameDicePlayerOperationRoomNotify.cs
/// <summary>
/// 
/// </summary>
namespace org.zgl.service.client.dice{
	[Proxy(gameId = -12)]
	public interface GameDicePlayerOperationRoomNotify : INotify { 
		/// <summary>
		/// 
		/// </summary>
		void end();
		/// <summary>
		/// 
		/// </summary>
		void start();
		/// <summary>
		/// 结算是财富更新
		/// </summary>
		void settleAccountsWeathUpdate(long betGold,long winGold,long residueGold,int betPosition,float rate);
		/// <summary>
		/// 结算是财富更新
		/// </summary>
		void positionPlayerWeathUpdate(long uid,long exchangeGold,long residueGold);
		/// <summary>
		/// 结算排行
		/// </summary>
		void settleAccountRanking(int count1,int count2,DiceSettleRankingListDto dto);
		/// <summary>
		/// 收到聊天信息
		/// </summary>
		void receiveChatMsg(ChatDto msg);
		/// <summary>
		/// 有人上位置
		/// </summary>
		void playerPositionUp(GameDicePositionPlayerInfoDto dto);
		/// <summary>
		/// 房间有玩家财富变更同步更新，uid：变更得uid，dto：变更得财富
		/// </summary>
		void playerWeathUpdate(long uid,ItemListDto dto);
		/// <summary>
		/// 有玩家清除下注
		/// </summary>
		void playerClearBet(long uid,long residueGold,long roomAllGold,int nowBetCount);
		/// <summary>
		/// 踢人下坐,selfUserName：踢人的用户名，targetUserName：被踢的人的用户名，targetUid：被踢的人的uid
		/// </summary>
		void kicking(string selfUserName,string targetUserName,long targetUid);
		/// <summary>
		/// 
		/// </summary>
		void stopBet();
		/// <summary>
		/// 
		/// </summary>
		void playerSettle(DiceSettleRankingDto d);
		/// <summary>
		/// 有人下位置
		/// </summary>
		void playerPositionDown(long uid);
	}
}

File: componet/src/main/java/org/zgl/service/client/dice/GameDicePlayerOperationRoomNotify.java
package org.zgl.service.client.dice;

import org.zgl.dto.ItemListDto;
import org.zgl.dto.clinet.commond.ChatDto;
import org.zgl.dto.clinet.dice.DiceSettleRankingDto;
import org.zgl.dto.clinet.dice.DiceSettleRankingListDto;
import org.zgl.dto.clinet.dice.GameDicePositionPlayerInfoDto;
import org.zgl.proxy.MethodDesc;
import org.zgl.proxy.rule.INotify;

/**
 * @author： 猪哥亮
 * @创建时间： 2018/8/18
 * @文件描述：
 */
public interface GameDicePlayerOperationRoomNotify extends INotify {
    @MethodDesc("房间有玩家财富变更同步更新，uid：变更得uid，dto：变更得财富")
    void playerWeathUpdate(long uid, ItemListDto dto);
    @MethodDesc("结算是财富更新")
    void settleAccountsWeathUpdate(long betGold, long winGold, long residueGold, int betPosition, float rate);
    @MethodDesc("结算是财富更新")
    void positionPlayerWeathUpdate(long uid, long exchangeGold, long residueGold);
    @MethodDesc("有玩家清除下注")
    void playerClearBet(long uid, long residueGold, long roomAllGold, int nowBetCount);
    @MethodDesc("有人上位置")
    void playerPositionUp(GameDicePositionPlayerInfoDto dto);
    @MethodDesc("有人下位置")
    void playerPositionDown(long uid);
    @MethodDesc("踢人下坐,selfUserName：踢人的用户名，targetUserName：被踢的人的用户名，targetUid：被踢的人的uid")
    void kicking(String selfUserName, String targetUserName, long targetUid);
    @MethodDesc("结算排行")
    void settleAccountRanking(int count1, int count2, DiceSettleRankingListDto dto);
    void start();
    void stopBet();
    void end();
    @MethodDesc("收到聊天信息")
    void receiveChatMsg(ChatDto msg);
    void playerSettle(DiceSettleRankingDto d);
}
